### ISSUE TITLE
Sets are broken

### ISSUE THREAD

**Issue by stathissideris** [stathissideris/spec-provider#8]
```clojure
> (pprint-specs
    (infer-specs
      [#{:a}] :foo/stuff) *ns* 's)

(s/def :foo/stuff (s/or))
```

**Comment by stathissideris** [comment]
Fixed, you now get:
```clojure
(s/def :foo/stuff (s/coll-of keyword? :kind set?))
```

**Comment by stathissideris** [comment]
Fix released in 0.4.4

**Comment by mfikes** [comment]
Sorry for the noise above (I had mistakenly referred to this ticket in a commit message).
